pkg/services/tree/service.go

```diff
@@ -483,13 +485,16 @@ func (s *Service) Apply(_ context.Context, req *ApplyRequest) (*ApplyResponse, e
 		return nil, fmt.Errorf("can't parse meta-information: %w", err)
 	}
 
-	d := pilorama.CIDDescriptor{CID: cid, Position: pos, Size: size}
-	resp := &ApplyResponse{Body: &ApplyResponse_Body{}, Signature: &Signature{}}
-	return resp, s.forest.TreeApply(d, req.GetBody().GetTreeId(), &pilorama.Move{
-		Parent: op.GetParentId(),
-		Child:  op.GetChildId(),
-		Meta:   meta,
-	}, false)
+	s.replicateLocalCh <- applyOp{
+		treeID:        req.GetBody().GetTreeId(),
+		CIDDescriptor: pilorama.CIDDescriptor{CID: cid, Position: pos, Size: size},
+		Move: pilorama.Move{
+			Parent: op.GetParentId(),
+			Child:  op.GetChildId(),
+			Meta:   meta,
+		},
+	}
+	return &ApplyResponse{Body: &ApplyResponse_Body{}, Signature: &Signature{}}, nil
```

Comment:
So the caller won't know if `TreeApply` is to fail?

Reply:
Yes, it is not important:
1. The caller has no real options to handle this error.
2. Background synchronization will do it's job and sync any missing operation.